### PR TITLE
Create FibonacciMultiStock.py

### PR DESCRIPTION
**Pull request by aidanvillasenor**

### CODE REVIEW

**Comment by echavemann** [comment]
Can we switch inputs to be **kwargs? Look at other dev algos for reference.

**Review by echavemann — CHANGES_REQUESTED** [review]
kwargs pls

**Comment by echavemann** [comment]
also its called multi but its single

**Comment by HZZX0318** [comment]
Updated to multi and with **kwargs

File: Strats/Fibonacci/FibonacciMultiStock.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                for band in self.bands[ticker]:
+                    if prev <= band and price >= band:
+                        for i in range(self.ordersWhenHitsBand):
+                            #if before we declined, then started going up and bounced off a band, add a sell order                      
+                            self.orders[ticker].append("SELL")
+        self.data[ticker].append(price)
+        return self.orders
+        
+
+    def updateAll(self, newData):
```
Comment (echavemann) [inline Strats/Fibonacci/FibonacciMultiStock.py:116]: please describe new data with a docstring and rename this function to update

File: Strats/Fibonacci/FibonacciMultiStock.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+import collections
+import math
+
+#Very simple multi-stock template. We will be generating these to get started with a simple strategy, and then generalizing it outward.
+class FibonacciMultiStock():
+    def __init__(self, scope = 900, ordersWhenHitsBand = 1, **kwargs):
```
Comment (echavemann) [inline Strats/Fibonacci/FibonacciMultiStock.py:7]: describe the requisite kwargs please

**Review by echavemann — CHANGES_REQUESTED** [review]

File: Strats/Fibonacci/FibonacciMultiStock.py (hunk 1 above)
Comment (HZZX0318) [reply Strats/Fibonacci/FibonacciMultiStock.py]: Done

File: Strats/Fibonacci/FibonacciMultiStock.py (hunk 2 above)
Comment (HZZX0318) [reply Strats/Fibonacci/FibonacciMultiStock.py]: Done

**Review by echavemann — APPROVED** [review]

**Pull request merged**